docs/releases/5.1.txt

```diff
@@ -170,6 +170,9 @@ Database backends
 * ``"pool"`` option is now supported in :setting:`OPTIONS` on PostgreSQL to
   allow using :ref:`connection pools <postgresql-pool>`.
 
+* ``"pool"`` option is now supported in :setting:`OPTIONS` on Oracle to
+  allow using :ref:`connection pools <oracle-pool>`.
```

Comment:
Again, "session" not connection pools.

Reply:
See above: the driver terminology is 'connection pool'.